prqlc/prqlc/src/cli/mod.rs

```diff
@@ -199,6 +211,37 @@ pub struct IoArgs {
     main_path: Option<String>,
 }
 
+#[derive(Copy, Clone, Debug, Default)]
+struct LoggingHelp;
+
+impl LogLevel for LoggingHelp {
```

Comment:
This is useful!

Reply:
I refactored this out into a separate PR #4471